src/labeler.ts

```diff
@@ -97,10 +100,21 @@ async function getLabelGlobs(
   client: ClientType,
   configurationPath: string
 ): Promise<Map<string, StringOrMatchConfig[]>> {
-  const configurationContent: string = await fetchContent(
-    client,
-    configurationPath
-  );
+  let configurationContent: string;
+
+  try {
+    configurationContent = (
+      await readFile(configurationPath, {
+        encoding: 'utf8'
+      })
+    ).toString();
+  } catch (error) {
+    core.info(
+      `${warningPrefix} configuration file (path: ${configurationPath}) not found locally (${error}), fetching via the api`
```

Comment:
I wouldn't consider this as a warning.

Reply:
@AndreiLobanovich The warning prefix makes this look stronger than it should IMO, even if logged with info severity. Is it necessary to have the warning prefix?

Reply:
@Youssef1313, the thing is - users were having a hard time understanding what's wrong when config file was misconfigured. info-warning should address that.

Reply:
@lrstanley could you please pull the latest changes from the `main` branch? We merged the [pull request](https://github.com/actions/labeler/pull/475) that improves error message handling. With these changes, the warning prefix is no longer needed so please remove it.